docs/mobile.md

```diff
@@ -12,22 +13,25 @@ By default, the application binds to `localhost`. These instructions bind to `0.
 
 1. Find your Local Area Network IP address. On a MacBook, this is available at **System Preferences → Network**. The address may start with `192.168`.
 
-2. To your app's `application.yml` file, add the below. Fill in your actual LAN IP address. The final line creates a **confirm now** link in place of email confirmation.
+2. In your app's `application.yml` file, add the below. Be sure to indent these lines and include them in the `development:` section. Also, fill in your actual LAN IP address. The final line creates a **confirm now** link in place of email confirmation.
+
 ```yaml
-domain_name: 192.168.x.x:3000
-mailer_domain_name: 192.168.x.x:3000
-enable_load_testing_mode: true
+development:
+  domain_name: 192.168.x.x:3000
+  mailer_domain_name: 192.168.x.x:3000
+  enable_load_testing_mode: true
 ```
 
 3. In the Chrome web browser of your development computer, visit `chrome://inspect`
 
 4. Click on **Port forwarding**. For port `8234` enter `0.0.0.0:3000`. Check **Enable port forwarding** and click **Done**. These screenshots illustrates enabling port forwarding on a MacBook:
 
-Click on Port forwarding   |  Enter IP and enable
-:-------------------------:|:-------------------------:
-![port-forwarding-button](https://user-images.githubusercontent.com/546123/231608927-5f577e1a-bc82-47c6-b69a-d592c551a99f.png)  |  ![port-forwarding-settings](https://user-images.githubusercontent.com/546123/231608489-f09f281e-305d-4200-9f21-9d772773a113.png)
+|                                                    Click on Port forwarding                                                    |                                                       Enter IP and enable                                                        |
```

Comment:
can you add a screen shot for this change? i want to understand what the visual impact is of updating this formatting

Reply:
Sure, here is the before.
![Screen Shot 2023-06-22 at 3 16 02 PM](https://github.com/18F/identity-idp/assets/80347702/87bff58c-a16d-480a-9a29-5b4d6da8097e)

Reply:
And here is the after.
![Screen Shot 2023-06-22 at 3 17 58 PM](https://github.com/18F/identity-idp/assets/80347702/bfc48b5b-103c-4587-b59a-186296c5cffd)